src/NativeScript/CMakeLists.txt

```diff
@@ -195,6 +195,12 @@ set_target_properties(NativeScript PROPERTIES
     "
 )
 
+add_custom_command(TARGET NativeScript
```

Comment:
I think it would be better to create a new custom target and add a dependency to it rather than use a custom build command on the runtime target.